src/access/AccessControl.cpp

```diff
@@ -629,9 +635,17 @@ void AccessControl::NotifyEntryChanged(const SubjectDescriptor * subjectDescript
     }
 }
 
+AccessControl * GetDefaultAccessControl()
+{
+    static AccessControl sInstance;
+    return &sInstance;
+}
+
+AccessControl * globalAccessControl = nullptr;
+
 AccessControl & GetAccessControl()
 {
-    return *globalAccessControl;
+    return *(globalAccessControl ?: GetDefaultAccessControl());
```

Comment:
AFAIK, omitting the argument between ? and : is a GNU extension. Please follow the C++ standard.